standard.md

```diff
@@ -0,0 +1,94 @@
+# The definition of the standard
+
+## Glossary
+
+There are three levels of definitions in this standard:
+
+- **Must**: all diagnostic have to meet this conditions to comply with the standard.
+- **Recommend**: diagnostics are encouraged to meet them but they are not mandatory. Nevertheless, if a list of compatible diagnostics is compiled, those who will meet them will be highlighted.
+- **Suggest**: they will never ve enforced, but the team consider them as nice to have. They mostly correspond to quality of life improvements.
+
+In the same way, when defining the content of the yaml files, three different levels of definitions are  used:
+
+- **Required**: they must allways be present with the defined meaning and allowed values.
+- **Reserved**: they can be omitted, but in the case that are present, they must have the defined meaning and allowed values.
+- **Custom**: in all files, extra options can be added as needed by the diagnostics or the tools. The only requirement is that they do not collide with any `required` or `reserved` option.
```

Comment:
Can we add that custom keys type is `str` ?

Reply:
I don't think so. One of the advantages of using yaml is that it directly supports numbers and boolean values, and I think lot's of custom keys will be numeric thresholds and boolean toggles

Reply:
@senesis does this explanation convince you?

Reply:
Fine